File: system-test/dotnet/SystemTests/Legacy/Mod10/AcceptanceTests/Base/BaseAcceptanceTest.cs
using Dsl.Port;
using Dsl.Core.Shared;
using SystemTests.Legacy.Mod10.Base;

namespace SystemTests.Legacy.Mod10.AcceptanceTests.Base;

public abstract class BaseAcceptanceTest : BaseScenarioDslTest
{
    protected override ExternalSystemMode? GetFixedExternalSystemMode()
    {
        return ExternalSystemMode.Stub;
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod10/acceptance/base/BaseAcceptanceTest.java
package com.mycompany.myshop.systemtest.legacy.mod10.acceptance.base;

import com.mycompany.myshop.systemtest.legacy.mod10.base.BaseScenarioDslTest;
import com.mycompany.myshop.systemtest.configuration.ExternalSystemMode;

public abstract class BaseAcceptanceTest extends BaseScenarioDslTest {
    @Override
    protected ExternalSystemMode getFixedExternalSystemMode() {
        return ExternalSystemMode.STUB;
    }
}



